includes/class-create-block-theme-admin-landing.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * The wp-admin landing page for the Create Block Theme plugin.
+ * @since      2.2.0
+ * @package    Create_Block_Theme
+ * @subpackage Create_Block_Theme/includes
+ * @author     WordPress.org
+ */
+class CBT_Admin_Landing {
+
+	public function __construct() {
+		add_action( 'admin_menu', array( $this, 'create_admin_menu' ) );
+	}
+
+	function create_admin_menu() {
+		if ( ! wp_is_block_theme() ) {
+			return;
+		}
+
+		$landing_page_slug       = 'create-block-theme-landing';
+		$landing_page_title      = _x( 'Create Block Theme', 'UI String', 'create-block-theme' );
+		$landing_page_menu_title = $landing_page_title;
+		add_theme_page( $landing_page_title, $landing_page_menu_title, 'edit_theme_options', $landing_page_slug, array( 'CBT_Admin_Landing', 'admin_menu_page' ) );
+
+	}
+
+	public static function admin_menu_page() {
+
+		$asset_file = include plugin_dir_path( __DIR__ ) . 'build/admin-landing-page.asset.php';
+
+		// Enqueue CSS dependencies of the scripts included in the build.
+		foreach ( $asset_file['dependencies'] as $style ) {
+			wp_enqueue_style( $style );
+		}
+
+		// Enqueue CSS of the app
+		wp_enqueue_style( 'create-block-theme-app', plugins_url( 'build/admin-landing-page.css', __DIR__ ), array(), $asset_file['version'] );
```

Comment:
```suggestion
		// Enqueue CSS of the app
		wp_enqueue_style( 'create-block-theme-app', plugins_url( 'build/admin-landing-page.css', __DIR__ ), array( 'wp-components' ), $asset_file['version'] );
```

`$asset_file['dependencies']` also includes unnecessary items such as `react` and `wp-api-fetch`. The only required dependency should be `wp-components`.

Additionally, wouldn't `admin_enqueue_scripts` hook be more appropriate for enqueuing CSS and JS?

Reply:
Why wouldn't react and wp-api-fetch (and anything else in the file) be required?

Reply:
There was a slight error in my explanation. `$asset_file['dependencies']` contains the handle names of dependent _script_ libraries. It is not a style handle name. So I don't think this array is suitable for `wp_enqueue_style` dependencies.